Service/src/test/java/com/epam/esm/validator/GiftCertificateValidatorTest.java

```diff
@@ -0,0 +1,71 @@
+package com.epam.esm.validator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class GiftCertificateValidatorTest {
+    private GiftCertificateValidator validator;
+
+    @BeforeEach
+    void setUp(){
+        validator = new GiftCertificateValidator();
```

Comment:
Why new validator should be created before each test?

Reply:
Corrected